multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -359,12 +312,16 @@ write_files:
                       "-initial-cluster-token",
                       "skydns-etcd"
                     ],
-                    "image": "gcr.io/google_containers/etcd:2.0.9",
+                    "image": "gcr.io/google_containers/etcd:2.2.1",
```

Comment:
Let's add the arch back, given that's what they are using upstream. I believe they are eventually deprecating their non-arch builds (see: https://github.com/kubernetes/kubernetes/blob/master/cluster/images/etcd/Makefile#L56)